frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -387,6 +387,15 @@ function FileManagerMenu:setUpdateItemTable()
             end,
         })
     end
+    if Device:isAndroid() then
+        table.insert(self.menu_items.developer_options.sub_item_table, {
+            text = _("Start Eink Test"),
```

Comment:
I believe it's spelled E-ink in the rest of the program. (Please ignore if I recall incorrectly. ;-) )
```suggestion
            text = _("Start E-ink Test"),
```